### DISCUSSION TITLE
Opinion on Kaito Icons

### DISCUSSION THREAD

**Discussion by Goldenfreddy0703** [Goldenfreddy0703/plugin.video.kaito.beta#44] [General]
Hey @Gujal00 , sorry for bothering you but i need your opinion, i've been thinking of adding icons to kaito but i have no idea how which is why i've been using autowidget to help me on that. What do you think of these icons? 
[Kaito Icons.zip](https://github.com/Goldenfreddy0703/plugin.video.kaito.beta/files/9344035/Kaito.Icons.zip)

- Each menu item you see matches its icon
- All these are made from Photoshop
- These are free to use
- There is a color and white version. 
- I have made a pack of these on my repository.

**Comment by Gujal00** [comment]
Yeah good idea to see icons instead of the addon icon everywhere. I will see how it can be added into the addon.

**Reply by Goldenfreddy0703** [reply]
That would be splendid, i will send a few screenshots of what it would looks like.

**Reply by Goldenfreddy0703** [reply]
Main Menu:
![Main Menu](https://user-images.githubusercontent.com/62456796/184811639-384ce3f2-7fb7-41d7-af4d-ebfc983b6edc.png)
AniList:
![AniList](https://user-images.githubusercontent.com/62456796/184811660-a672fc90-cce3-47b4-8a1d-2ff27243331a.png)
Kitsu:
![Kitsu](https://user-images.githubusercontent.com/62456796/184811664-80e47ccf-f36d-4321-877f-3f8fb78ef845.png)
Mal:
![Mal](https://user-images.githubusercontent.com/62456796/184811674-8752e3fb-2d4b-4a0c-82d1-feda0cd6a417.png)
Tools:
![Tools](https://user-images.githubusercontent.com/62456796/184811682-114d12e6-2150-424c-9427-e73f3c7e2706.png)

Ok here is a preview of the screenshots.
 - On the Main Menu, ignore the Tmdb Anime menu item. I just added that from tmdb helper and made anime sections for them.

**Comment by Gujal00** [comment]
So you mean like this Main menu?
![screenshot00000](https://user-images.githubusercontent.com/17394059/184817673-6f3cb392-2e19-411f-9902-8131ecdd9b22.png)

**Reply by Goldenfreddy0703** [reply]
Yes

**Reply by Goldenfreddy0703** [reply]
It looks really great!!!

**Reply by Gujal00** [reply]
Okay I need a "Next" coloured icon, the one currently in images folder is a transparency one, so more like the white ones you have

So main menu is done and working as above screenshot. Now have to go through each menu and get it done :)

**Reply by Goldenfreddy0703** [reply]
Ahh ok, if there is a specific icon that is missing or does not look right, tell me and I will try my best to make it look great.

**Reply by Gujal00** [reply]
Also some icons are identical, is that intentional?
popular this season = all time popular
Latest = Latest English dubbed = upcoming next season
current = currently watching

**Reply by Goldenfreddy0703** [reply]
Yes cause they match the label of each menu item.

**Reply by Gujal00** [reply]
other menus done. dont have kitsu so cant see it, it should work
![screenshot00000](https://user-images.githubusercontent.com/17394059/184837340-0f832140-3370-49a0-8385-bba2ce8fd4c8.png)
![screenshot00000](https://user-images.githubusercontent.com/17394059/184840106-698d2eca-1067-48b9-9628-0b353036a13f.png)
![screenshot00001](https://user-images.githubusercontent.com/17394059/184840132-1b5393a7-28f0-405c-ba20-9641f7980854.png)

**Reply by Goldenfreddy0703** [reply]
Oh hey that looks really nice. I love them!!!

**Reply by Gujal00** [reply]
Added option in settings to select coloured or white theme
![screenshot00000](https://user-images.githubusercontent.com/17394059/184843821-73bc6a84-d45f-45c0-ac1c-6a25435a284f.png)

**Reply by Goldenfreddy0703** [reply]
Hey one thing I accidentally forgot but on Kaito, when you click search on the main menu, it shows two menu items that say new search and clear search history, could you add icons for those? I believe I have them labeled and I forgot about that.

**Reply by Goldenfreddy0703** [reply]
> Added option in settings to select coloured or white theme
> ![screenshot00000](https://user-images.githubusercontent.com/17394059/184843821-73bc6a84-d45f-45c0-ac1c-6a25435a284f.png)
> 
> 

Oh niceeee, thank you very much for adding an option for that. That's really cool.

**Reply by Gujal00** [reply]
Will do. You can test run this if you switch to feature/add-art branch in git

**Reply by Goldenfreddy0703** [reply]
Oh nice, ok I will test it and let you know the results.

**Reply by Goldenfreddy0703** [reply]
Hey, so i just tested it out with the colored and white icons and they are amazing. One little small problem i notice was that we would need to also change the icon and poster from looking at it. 
![image](https://user-images.githubusercontent.com/62456796/184848399-552635b1-5d78-4aa7-b516-3b30e24f9f27.png)
If you could add icon and thumb to the art, that would be great. 
Thank you

**Reply by Gujal00** [reply]
Search menu done, pull the branch again if you haven't
![screenshot00001](https://user-images.githubusercontent.com/17394059/184848351-59111156-17db-4b5e-8494-9bfa91241853.png)

**Reply by Gujal00** [reply]
where is the icon and thumb visible, i only tested in the default skin

**Reply by Goldenfreddy0703** [reply]
Some skins like Artic Horizon 2 or Auramod usually require the thumbnail or icon

**Reply by Goldenfreddy0703** [reply]
For example: this is what i see in Artic Horizon 2. 
![image](https://user-images.githubusercontent.com/62456796/184849887-e75a9385-e94c-4400-8172-b80886582912.png)
If we can point these icons to icon and thumb, that would help users when changing viewtypes so that they don't have to see the kaito logo.

**Reply by Goldenfreddy0703** [reply]
> Search menu done, pull the branch again if you haven't ![screenshot00001](https://user-images.githubusercontent.com/17394059/184848351-59111156-17db-4b5e-8494-9bfa91241853.png)

Oh yeah, i tested it out and it looks fantastic, thank you!!!

**Reply by Gujal00** [reply]
Okay pull branch and test again with the other skins, i am now setting poster, thumb and icon for all items

**Reply by Goldenfreddy0703** [reply]
Oh ok, while you were looking at the other skin. I tried coding the menu items just to see if i got it correct and i failed lol 😂
![image](https://user-images.githubusercontent.com/62456796/184854482-da62ce88-01aa-4e8b-8bab-88ed99f60f5a.png)

**Reply by Goldenfreddy0703** [reply]
Ok so from looking at it, everything looks good so far, the rest just thumb and icon for all items but so far, everything is now looking great. 
![image](https://user-images.githubusercontent.com/62456796/184855281-dee6c40b-69e7-4bc7-81d3-57bbb663309a.png)
AniList, Kitsu, and Mal need there thumb and icon as well but everything else is looking pretty good.

**Reply by Gujal00** [reply]
I did add icon and thumb to anilist, kitsu and mal. Is it not working?
Please pull again and check if the thumb and icon are there for Anilist, Kitsu and Mal

**Reply by Goldenfreddy0703** [reply]
I don't think so, let me restart and check again, its probably on my end

**Reply by Gujal00** [reply]
Also remember to make a coloured next icon, currently i have put the white next icon into the coloured folder

**Reply by Goldenfreddy0703** [reply]
oh definitely, i will in a little bit

**Reply by Gujal00** [reply]
> I don't think so, let me restart and check again, its probably on my end

Ah my fault, I committed the change but did not push to repo. Please pull again and test

**Reply by Goldenfreddy0703** [reply]
![image](https://user-images.githubusercontent.com/62456796/184857881-8e68cc6e-cc1e-475a-a7f5-0dd9bb665664.png)
Ok so i clear cache, rebuild database, deleted my icons folder and tried it again and i just get the kaito logo for AniList, Kitsu, and Mal.

**Reply by Gujal00** [reply]
Yeah you need to pull the branch from git, and test please

**Reply by Goldenfreddy0703** [reply]
![image](https://user-images.githubusercontent.com/62456796/184858142-88958517-1c16-4b44-8d0a-b37d36ec725e.png)
from looking at the artwork, you did get the poster, just not the icon or thumb. Let me take a look at your code to see how you added poster for AniList, Kitsu, and Mal.

**Reply by Goldenfreddy0703** [reply]
I just found out you made a pull 8 min ago, let me test that again real quick

**Reply by Goldenfreddy0703** [reply]
Ok you fixed them, just tested it and it works now

**Reply by Goldenfreddy0703** [reply]
![image](https://user-images.githubusercontent.com/62456796/184859000-fd1a5a57-6192-4a5e-a3f3-9cf4c478d7c8.png)
Niceee, everything is working now and it looks great, thank you very much. 

Im going to get started on the next icon now

**Reply by Goldenfreddy0703** [reply]
![next](https://user-images.githubusercontent.com/62456796/184860196-a007ffef-b741-47f5-b208-b3e4599f5230.png)
Hey so what do you think of the next.png so far.

**Reply by Gujal00** [reply]
> Hey so what do you think of the next.png so far.

The current next icon does not have 'kaito' in it. DO you want this going forward? if so make both coloured and white versions

**Reply by Goldenfreddy0703** [reply]
![next](https://user-images.githubusercontent.com/62456796/184860939-79c3feb7-8a27-4b6c-8132-55e8b932519a.png)
Here is also a white png as well

**Reply by Goldenfreddy0703** [reply]
> > Hey so what do you think of the next.png so far.
> 
> The current next icon does not have 'kaito' in it. DO you want this going forward? if so make both coloured and white versions

mmm good question, ok well sense i made a next for seren, i might as well use the new next icons so im going forward with this.

**Reply by Gujal00** [reply]
okay the new icons are now tinified and added to the feature branch

**Reply by Goldenfreddy0703** [reply]
Niceeee!!!!

**Reply by Gujal00** [reply]
Enough coding for the day, time to bed, almost 11pm here now 😂

**Reply by Goldenfreddy0703** [reply]
![image](https://user-images.githubusercontent.com/62456796/184862617-56e0e420-d2bd-407e-b3d4-33c189621f68.png)
Ok this looks fantastic, i love it, You did a really good job with this and im really happy to have you contribute to this. You are the best!!!

**Reply by Goldenfreddy0703** [reply]
> Enough coding for the day, time to bed, almost 11pm here now 😂

Lol, its already 6am over here, you have a goodnight and again, thank you for all the help you provided on this.
